app/views/capacity.html

```diff
@@ -174,6 +187,20 @@ <h3 class="heading-medium">Capacity breakdown</h3>
           <td headers="cms breakdown">{{ member.grades[i].cmsPercentage | round(1) }}%</td>
         </tr>
         {% endfor %}
+        {% if loop.last %}
+        <tr>
+            <td colspan="2" headers="name org">{{ member.name }}</td>
+            <td headers="capacity breakdown">{{ member.capacity | round(1) }}%</td>
+            <td headers="cases breakdown">{{ member.totalCases | round(1) }}</td>
+            <td headers="cases breakdown">{{ member.totalTotalT2aCases | round(1) }}</td>
```

Comment:
Header include `t2aCases` as opposed to `cases`? Here and on line 138

Reply:
Ah balls, good spot. Will get that in.